v0/destinations/singular/util.js

```diff
@@ -138,6 +139,26 @@ const platformWisePayloadGenerator = (message, isSessionEvent) => {
     } else {
       payload.dnt = 1;
     }
+
+    // Singular accepts install_time & update_time values in Unix timestamps format,
+    // hence we are converting it
+    const it =
+      message.properties?.install_time ||
+      message.originalTimestamp ||
+      message.timestamp;
+    const ut =
+      message.properties?.update_time ||
+      message.originalTimestamp ||
+      message.timestamp;
+    payload.install_time = toUnixTimestamp(it);
+    payload.update_time = toUnixTimestamp(ut);
+
+    // set openuri in payload as "" (empty string) if url is not passed in properties
+    if (!message.properties.url) {
+      payload.openuri = "";
+    } else {
+      payload.openuri = message.properties.url;
+    }
```

Comment:
```suggestion
    payload.openuri = message.properties.url || "";
```

Reply:
Similarly can we check other fields as well?
<img width="996" alt="Screenshot 2022-10-12 at 10 44 51 AM" src="https://user-images.githubusercontent.com/25389167/195255880-6a9a11a2-1e37-4d1d-805d-7ce750244e78.png">

Reply:
We are removing all null and undefined values from the payload in the end.